packages/quantic/force-app/main/default/lwc/searchBoxStyle/searchBoxStyle.css

```diff
@@ -31,18 +31,20 @@
   height: 48px;
 }
 
-.searchbox__clear-button {
```

Comment:
there was duplicates `searchbox__clear-button` classes here so moved the css into 1